pages/test.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
```

Comment:
I see you made a separate page for testing purposes, but you can directly add the `<Passports />` under `index.tsx` in place of `<Staff/>`
<img width="500" alt="image" src="https://user-images.githubusercontent.com/62922309/228150256-da841b03-6885-4e1a-8467-f11d6fcaa85d.png">